MapboxNavigation/BottomBannerViewController.swift

```diff
@@ -21,6 +21,16 @@ public protocol BottomBannerViewControllerDelegate: class {
 @objc(MBBottomBannerViewController)
 open class BottomBannerViewController: UIViewController, NavigationComponent {
     
+    /*
+     A padded margin that covers the bottom safe area of the device, if any.
+     */
+    lazy open var bottomPaddingView: BottomBannerView = .forAutoLayout()
```

Comment:
It’s a bit confusing to see a view described as a margin. Can we clarify the comment somehow, perhaps by describing it as a spacer that simulates the margin?